pywebhdfs/webhdfs.py

```diff
@@ -77,8 +83,17 @@ def create_file(self, path, file_data, **kwargs):
 
         # make the initial CREATE call to the HDFS namenode
         optional_args = kwargs
-        uri = self._create_uri(path, operations.CREATE, **optional_args)
-        init_response = requests.put(uri, allow_redirects=False)
+
+        hosts = self._resolve_federation(path)
+        for host in hosts:
+            uri = self._create_uri(host, path, operations.CREATE,
+                                   **optional_args)
+            init_response = requests.put(uri, allow_redirects=False)
+            if not _is_standby_exception(init_response):
+                _move_active_host_to_head(hosts, host)
+                break
+        else:
+            raise errors.PyWebHdfsException(msg="Could not find active host")
```

Comment:
I plan to make `ActiveHostNotFound` exception (relate HA) and `CorrespondHostsNotFound` exception (relate federation).

Then I want to know USECACE of `_raise_pywebhdfs_exception()`.
And Should I use this method to raise above exceptions?